### PR TITLE
Add deprecation messages to all messages now in example_interfaces

### PR DESCRIPTION
**Pull request by tfoote**
Followup to #89

### CODE REVIEW

File: std_msgs/msg/UInt8MultiArray.msg [hunk 1]
```diff
@@ -1,3 +1,8 @@
+# This was originally provided as an example message.
+# It is deprecated as of Foxy, it is recommended to create your own
```
Comment (hidmic) [inline std_msgs/msg/UInt8MultiArray.msg:2]: @tfoote consider
```suggestion
# It is deprecated as of Foxy. 
# It is recommended that you create your own
```
same elsewhere.

**Comment by tfoote** [comment]
It's been in ROS 1 that we generally recommend against using almost any message in std_msgs except Header for a long time for stability.
From the Overview: http://wiki.ros.org/std_msgs

```
However, these types do not convey semantic meaning about their contents: every message simply has a field called "data". Therefore, while the messages in this package can be useful for quick prototyping, they are NOT intended for "long-term" usage. For ease of documentation and collaboration, we recommend that existing messages be used, or new messages created, that provide meaningful field name(s).

Note that this package also contains the "MultiArray" types, which can be useful for storing sensor data. However, the same caveat applies: it's usually "better" (in the sense of making the code easier to understand, etc.) when developers use or create non-generic message types (see discussion in this thread for more detail).
```

However from the naming it leads to a lot of confusion.([For example](https://github.com/ros2/test_interface_files/pull/4#issuecomment-617790928)) And we want to take the opportunity in ROS 2 to avoid that ambiguity. The earlier we do it in this cycle the easier and less disruption it will cause.

File: std_msgs/msg/UInt8MultiArray.msg (hunk 1 above)
Comment (tfoote) [reply std_msgs/msg/UInt8MultiArray.msg]: Thanks that reads better, updated.

**Comment by jacobperron** [comment]
I guess we shouldn't have run CI for this change since it caused a test failure downstream: https://ci.ros2.org/view/nightly/job/nightly_linux_release/1548/testReport/ros2interface.src.ros2.ros2cli.ros2interface.test/test_cli/test_cli/

See a fix in https://github.com/ros2/ros2cli/pull/516

**Comment by ros-discourse** [comment]
This pull request has been mentioned on **ROS Discourse**. There might be relevant details there:

https://discourse.ros.org/t/soss-a-whole-new-approach-to-your-ros-1-ros-2-bridge/17040/18

**Comment by ros-discourse** [comment]
This pull request has been mentioned on **ROS Discourse**. There might be relevant details there:

https://discourse.ros.org/t/discussion-standardizing-ros-interfaces/33693/1

**Pull request merged**